Makefile

```diff
@@ -455,31 +456,26 @@ enterprise-code-checkin-test:
 test-pfs-server:
 	./etc/testing/pfs_server.sh $(ETCD_IMAGE) $(TIMEOUT)
 
-test-pfs-cmds:
-	@# Unlike test-pfs-server, this target requires a running cluster
-	go test ./src/server/pfs/cmds -count 1 -timeout $(TIMEOUT)
-
 test-pfs-storage:
 	go test ./src/server/pkg/storage/chunk -count 1 -timeout $(TIMEOUT)
 	go test ./src/server/pkg/storage/fileset/index -count 1 -timeout $(TIMEOUT)
 	go test ./src/server/pkg/storage/fileset -count 1 -timeout $(TIMEOUT)
 
-test-deploy-cmds:
-	go test ./src/server/pkg/deploy/cmds -count 1 -timeout $(TIMEOUT)
-
 test-config:
 	go test ./src/server/config -count 1 -timeout $(TIMEOUT)
 
-test-pps:
-	@# Travis uses the helper directly because it needs to specify a
-	@# subset of the tests using the run flag
-	@make RUN= test-pps-helper
-
-test-pps-helper: launch-stats launch-kafka docker-build-test-entrypoint
-	# Use the count flag to disable test caching for this test suite.
+test-pps: launch-stats launch-kafka docker-build-test-entrypoint
+	@# Use the count flag to disable test caching for this test suite.
 	PROM_PORT=$$(kubectl --namespace=monitoring get svc/prometheus -o json | jq -r .spec.ports[0].nodePort) \
-	  go test -v ./src/server -parallel 1 -count 1 -timeout $(TIMEOUT) $(RUN) && \
-	  go test -v ./src/server/pps/cmds -count 1 -timeout $(TIMEOUT)
+	  go test -v ./src/server -parallel 1 -count 1 -timeout $(TIMEOUT) $(RUN)
+
+test-cmds:
+	go install -v ./src/testing/match
+	go test -v ./src/server/pkg/deploy/cmds -count 1 -timeout $(TIMEOUT)
+	go test -v ./src/server/pfs/cmds -count 1 -timeout $(TIMEOUT)
+	go test -v ./src/server/pps/cmds -count 1 -timeout $(TIMEOUT)
+	@# TODO(msteffen) does this test leave auth active? If so it must run last
+	go test -v ./src/server/auth/cmds -count 1 -timeout $(TIMEOUT) 
```

Comment:
This is my bad, both for not suffixing the package with `cmds`, and for naming the test incorrectly, but would you mind adding config cmd tests here? The current target is [`test-config`](https://github.com/pachyderm/pachyderm/blob/30fedffbaccbac64427eb4bee4a8a2c491b04d13/Makefile#L464-L465).

Reply:
Done